src/Microsoft.AspNetCore.SignalR.Client/HubConnection.cs

```diff
@@ -45,29 +46,29 @@ public class HubConnection
             remove { _connection.Closed -= value; }
         }
 
+        public HubConnection(Uri url, IInvocationAdapter adapter)
```

Comment:
It was being tracked separately and yesterday I created individual issue for this - #235. We could fix it as part of this or separately.